public/index.html

```diff
@@ -0,0 +1,215 @@
+        <section class="featuredEls roaster" id="goldenCoffee">
+          <img src="{{featuredCoffeePic}}" alt="not found" />
+        </section>
+    </section>
+
+    <section id="mapHead">
+      <h1 id="mapH1">The matter of Weather</h1>
+      <img id ="mapDownArrow" class="arrows" src="/media/images/mapDownArrow.png" alt="arrow" />
+    </section>
+    <section class="coffeePages" id="coffeeMap">
+      <section id="mapContainer">
+        <ul id="mapMonths">
+          <li class="mapLi"><button class="mapButtons" data-content="january" id="january" type="button" name="button">january</button></li>
+          <li class="mapLi"><button class="mapButtons" data-content="february" id="february" type="button" name="button">february</button></li>
+          <li class="mapLi"><button class="mapButtons" data-content="march" id="march" type="button" name="button">march</button></li>
+          <li class="mapLi"><button class="mapButtons" data-content="april" id="april" type="button" name="button">april</button></li>
+          <li class="mapLi"><button class="mapButtons" data-content="may" id="may" type="button" name="button">may</button></li>
+          <li class="mapLi"><button class="mapButtons" data-content="june"id="june" type="button" name="button">june</button></li>
+          <li class="mapLi"><button class="mapButtons" data-content="july" id="july" type="button" name="button">july</button></li>
+          <li class="mapLi"><button class="mapButtons" data-content="august" id="august" type="button" name="button">august</button></li>
+          <li class="mapLi"><button class="mapButtons" data-content="september" id="september" type="button" name="button">september</button></li>
+          <li class="mapLi"><button class="mapButtons" data-content="october" id="october" type="button" name="button">october</button></li>
+          <li class="mapLi"><button class="mapButtons" data-content="november" id="november" type="button" name="button">november</button></li>
+          <li class="mapLi"><button class="mapButtons" data-content="december"id="december" type="button" name="button">december</button></li>
+        </ul>
+
+
+      <section id="mapBox" style="background-image: url({{map}})">
+        <canvas id="coffeeData" width="300" height="525"></canvas>
+      </section>
+      </section>
+    </section>
+    <section id="brewHead">
+      <h1 id="brewH1">How we Brew it</h1>
+      <img id ="straightArrowLeft" class="arrows" src="/media/images/straightArrowLeft.png" alt="arrow" />
+    </section>
+    <section class="coffeePages" id="brewMethods">
+      <section id="methodsContainer">
+        <section class="methodEls" id="brewImage">
+          <img src="{{brewPic}}" alt="not found" />
+        </section>
+        <section class="methodEls" id="brewSteps">{{brewInstructions}}</section>
+      </section>
+    </section>
+  </section>
+  </script>
+
+
+
+  </head>
+  <body>
+    <select id="coffeeCountry" name="Country">
```

Comment:
This dropdown could also use some additional styling. For how well the rest of the site looks, this needs a little love as well.